### PR TITLE
Document the block time in the `z_gettreestate` RPC response

### LINKED ISSUE #5913: Document the block time in the `z_gettreestate` RPC response
### Describe the issue

The function [z_gettreestate](https://github.com/zcash/zcash/blob/master/src/rpc/blockchain.cpp#L1225) doesn't mention the block time in its docs, even though the function returns it ([here](https://github.com/zcash/zcash/blob/master/src/rpc/blockchain.cpp#L1286)). The block time is then also not rendered in the public  [docs](https://zcash.github.io/rpc/z_gettreestate.html).

`lightwalletd` also expects the block time field [here](https://github.com/zcash/lightwalletd/blob/68789356fb1a75f62735a529b38389ef08ea7582/common/common.go#L108).

### PR DESCRIPTION
**Pull request by upbqdn**
Closes #5913.

### CODE REVIEW

**Review by steven-ecc — APPROVED** [review]
utACK

**Review by nuttycom — APPROVED** [review]
utACK

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -1233,7 +1233,8 @@ UniValue z_gettreestate(const UniValue& params, bool fHelp)
             "\nResult:\n"
             "{\n"
             "  \"hash\": \"hash\",         (string) hex block hash\n"
-            "  \"height\": n,            (numeric) block height\n"
+            "  \"height\": n,              (numeric) block height\n"
+            "  \"time\": n,                (numeric) block time\n"
```
Comment (daira) [inline src/rpc/blockchain.cpp:1237]: Please specify the format and epoch.

Comment (upbqdn) [reply src/rpc/blockchain.cpp:1237]: The PR got merged already, should I open another one?

Comment (steven-ecc) [reply src/rpc/blockchain.cpp:1237]: Sorry about that. If you don't mind, another PR would be great.

Comment (upbqdn) [reply src/rpc/blockchain.cpp:1237]: Sure, the new one is here https://github.com/zcash/zcash/pull/5916.

Comment (upbqdn) [reply src/rpc/blockchain.cpp:1237]: Resolved in https://github.com/zcash/zcash/pull/5916.

**Pull request merged**